packages/mask/src/extension/popups/pages/Personas/Accounts/index.tsx

```diff
@@ -19,7 +19,12 @@ const Accounts = memo(() => {
 
     const definedSocialNetworks = compact(
         getEnumAsArray(EnhanceableSite).map((x) => {
-            if (x.value === EnhanceableSite.Localhost || x.value === EnhanceableSite.OpenSea) return null
```

Comment:
you should add a new property called something like `canHasPersona` and filter by this property instead of doing this.

Reply:
I opened a new PR: #7511 to refactor this.